components/KeyFinancials.vue

```diff
@@ -29,16 +29,90 @@
       'ftsUrl': String,
     },
 
+    data() {
+      return {
+        ftsPlanId: 639,
```

Comment:
I can fix this in a follow-up, it has little to do with the current task of hooking FTS up to the site. The `asyncData` function just waits for both datasets to resolve so I can pull this value from Contentful in a second PR.